Tests/ApolloCacheDependentTests/LoadQueryFromStoreTests.swift

```diff
@@ -418,7 +418,7 @@ extension LoadQueryFromStoreTests {
           XCTAssertNil(result.errors, file: file, line: line)
           XCTAssertEqual(result.data?.hero?.name, "R2-D2", file: file, line: line)
           XCTAssertEqual(
-            Calendar.current.compare(expectedResultAge, to: result.context.resultAge, toGranularity: .minute),
```

Comment:
Minor: In those tests, you also don't need to use `Calendar` to compare `Date`

Reply:
Using the Calendar API to compare dates was something requested by the Apollo team, and these tests are from there.